java/com/google/javascript/jscomp/JsChecker.java

```diff
@@ -314,7 +319,11 @@ public CheckLevel level(JSError error) {
     return result.build();
   }
 
-  public static void main(String[] args) throws IOException {
+  static int processRequest(Collection<String> args) throws IOException {
+    String lastArg = Iterables.getLast(args, "");
+    if (lastArg.startsWith("@")) {
+      args = Files.readAllLines(Paths.get(lastArg.substring(1)), UTF_8);
+    }
```

Comment:
In theory if that happens, it means the prior arguments were meant for the worker and can be safely ignored. I'm assuming you saw [ExampleWorker](https://github.com/bazelbuild/bazel/blob/master/src/test/java/com/google/devtools/build/lib/worker/ExampleWorker.java)? I changed it to be more forgiving here because I saw some error message in Bazel that specifically talked about the `@` thing relating to the last parameter.